docs/source/en/advanced_tutorials/train_vit_using_pipeline_parallelism.md

```diff
@@ -2,8 +2,12 @@
 
 Author: Hongxin Liu, Yongbin Li
 
+**Prerequisite:**
```

Comment:
This tutorial is similar to vit using hybrid parallelism. Why not remove this one?

Reply:
updated